src/app/codegen-data-model-provider/tests/TestCodegenModelViaMocks.cpp

```diff
@@ -1267,6 +1319,62 @@ TEST(TestCodegenModelViaMocks, IterateOverGeneratedCommands)
     }
 }
 
+TEST(TestCodegenModelViaMocks, AcceptedCommandValidity)
+{
+    UseMockNodeConfig config(gTestNodeConfig);
+    CodegenDataModelProviderWithContext model;
+
+    // register a CHI on ALL endpoints
+    CustomListCommandHandler handler(chip::NullOptional, MockClusterId(1));
+    handler.SetHandleCommands(true);
+
+    handler.SetOverrideAccepted(true);
+    handler.AcceptedVec().push_back(1234);
+    handler.AcceptedVec().push_back(999);
```

Comment:
Why is 999 added here?

Reply:
Copied from other places. Removed now to make the test clean.